docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -37,7 +58,23 @@ Check for the below `INFO` log to confirm if the server started successfully,
 == Creating cache manager with clustering capabilities
 
 After starting the Terracotta server (Refer section: <<Starting Terracotta server>>), you can now proceed to create the cache manager.
```

Comment:
Could you change that to a link to the section as a hyper link around "starting the Terracotta server"? Also when doing such a link, please add an explicit session id.